test/test_operators.js

```diff
@@ -0,0 +1,150 @@
+////////////////////////////////////////////////////////////////////////////////
+// TEST CASE - EXPECT PASS
+////////////////////////////////////////////////////////////////////////////////
+// const x = 1;
+// x - 2;           // returns -1
+
+
+////////////////////////////////////////////////////////////////////////////////
+// TEST CASE - EXPECT FAIL
+////////////////////////////////////////////////////////////////////////////////
+const x = true;
```

Comment:
just nitpicking here, but this could be commented out

Reply:
But I wna test const declaration plus `true + 2`. If I comment this out, what does the test case look like? But in any case, pls feel free to update the test cases

Reply:
I saw that you commented out the other test cases as well so I thought you could comment it out for standardisation. It doesn't really matter in the long run, and I have another PR coming up with test cases in Jest.